robocop/checkers/lengths.py

```diff
@@ -151,9 +154,13 @@ class LineLengthChecker(RawFileChecker):
 
     def __init__(self):
         self.max_line_length = 120
+        # replace # noqa or # robocop, # robocop: enable, # robocop: disable=optional,rule,names
+        self.disabler_pattern = re.compile(r'(# )+(noqa|robocop: (?P<disabler>disable|enable)=?(?P<rules>[\w\-,]*))')
```

Comment:
Do we want to require space after `# robocop: ` ?
Maybe we should also accept `# robocop:disable`?

Reply:
Right, I think we could accept robocop:enable or robocop:disable as well - I will update disablers later to adjust it.